vulnerabilities/tests/test_nvd.py

```diff
@@ -139,9 +140,18 @@ def test_to_advisories(self):
                     [
                         Reference(
                             url="http://code.google.com/p/gperftools/source/browse/tags/perftools-0.4/ChangeLog",  # nopep8
+                            scores=[],
                         ),
                         Reference(
                             url="http://kqueue.org/blog/2012/03/05/memory-allocator-security-revisited/",  # nopep8
+                            scores=[],
+                        ),
+                        Reference(
+                            url="https://nvd.nist.gov/vuln/detail/CVE-2005-4895",  # nopep8
+                            scores=[
```

Comment:
Just a note on the naming consistency:
why would a list of `VulnerabilitySeverity` be called `scores`?
Either `VulnerabilitySeverity` should be renamed to `Score` which may be best.
Or `scores` should be renamed to `severities`

Reply:
That makes sense. 

I am going with  `scores-> severities`  since .

Reply:
Done